packages/core-components/src/components/DependencyGraph/types.ts

```diff
@@ -93,6 +94,38 @@ export namespace DependencyGraphTypes {
     props: RenderNodeProps<T>,
   ) => ReactNode;
 
+  /**
+   * Properties of {@link DependencyGraphTypes.RenderEdgeFunction} for {@link DependencyGraphTypes.DependencyEdge}
+   *
+   * @public
+   */
+  export type RenderEdgeProps<T = {}> = {
+    edge: T & {
+      points: { x: number; y: number }[];
+      label?: string;
+      labeloffset?: number;
+      labelpos?: string;
+      width?: number;
+      height?: number;
+      weight?: number;
+      minlen?: number;
+      showArrowHeads?: boolean;
+      from?: string;
+      to?: string;
+      relations?: string[];
+    };
+    id: dagre.Edge;
```

Comment:
how complex is this type? I think it would be good to hide the external dependency from api report

Reply:
It's not complex.   

```
export interface Edge {
    v: string;
    w: string;
    name?: string | undefined;
  }
```

I'll replace it.